using-timescaledb/backup.md

```diff
@@ -1,8 +1,22 @@
 # Backup & Restore [](backup)
 
-Backing up TimescaleDB takes advantage of the reliable functionality already available
-through PostgreSQL.  We recommend using one of two methods depending on your setup:
-`pg_dump` with `pg_restore` or the Write-Ahead Log (WAL).
+Backing up TimescaleDB takes advantage of the reliable functionality already
+available through PostgreSQL.  There are several ways to accomplish this,
+physical backups with `pg_basebackup` or another tool, or logical backups with
+`pg_dump` and `pg_restore`. Physical backups may also be used Write-Ahead Log
```

Comment:
I don't know what you mean here or "for which we offer some particular
integrations in containerized environments"